obspy/imaging/spectrum.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+"""
+Plotting the spectrum of seismograms
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA @UnusedWildImport
+import numpy as np
+import matplotlib.pyplot as plt
+from scipy import fft
+
+def plot_spectrum(data, samp_rate, outfile=None, fmt=None,
+             axes=None, title=None, show=True):
+
+    """
+     Compute and plot the spectrum of the input data
+     """
+
+    # enforce float for samp_rate
+    samp_rate = float(samp_rate)
+
+    npts = len(data)
+
+    # Compute the FFT
+    npts = len(data)
+    k = np.arange(npts)
+    T = npts/samp_rate
+    frq = k/T  # two sides frequency range
+    frq = frq[range(npts/2)]  # one side frequency range
+
+    X = fft(data)/npts  # fft computing and normalization
```

Comment:
Yea true...thats a bit awkward...one can also use `numpy.fft.rfft` but the scipy variant is faster.